app/Models/Story.php

```diff
@@ -69,9 +65,11 @@ protected static function booted()
             if ($story->user_id != $story->tester_id) {
                 // Check if user_id or tester_id has changed
                 if ($story->wasChanged('user_id') && $story->user_id) {
+                    //send email to the new developer 
```

Comment:
non controlli se l'utente collegato è developer, in generale prima di inviare le mail devi TOGLIERE la mail di chi è collegato